examples/uno/client/web/src/components/PlayerList/PlayerList.tsx

```diff
@@ -0,0 +1,26 @@
+import MiniCardsRow from "../MinICardCount/MiniCard";
+import { useHathoraContext } from "../../context/GameContext";
+
+export default function PlayerList() {
+  const { playerState, user, getUserName } = useHathoraContext();
+  return (
+    <div className="pt-5">
+      {playerState?.players.map((player) => (
+        <div
+          key={player.id}
+          className={`bg-slate-500 text-white shadow shadow-gray-600 p-3 rounded mb-2 ${
+            player.id === playerState?.turn ? "bg-green-600" : ""
+          }`}
+        >
+          {getUserName(player.id)} {player.id === user?.id ? "(You)" : ""}
```

Comment:
this seems to be getting called over and over again, resulting in repeated network calls to the coordinator

Reply:
@hpx7 addressed the issue it was due to an incorrect boolean check that was happening in the use effect in the context. App should only do a limit number of requests to fetch information